src/handwritten/task.py

```diff
@@ -1019,6 +1019,24 @@ def stop(self):
         """
         self._interpreter.stop_task(self._handle)
 
+    def wait_for_valid_timestamp(self, timestamp_event, timeout=10.0):
+        """
+        Wait until the specified timestamp has a value.
+
+        Use this VI to ensure the timestamp has a valid value to prevent an error when querying a timestamp value.
+
+        Args:
+            timestamp_event(nidaqmx.constants.TimestampEvent): Specifies the timestamp type to wait on. 
+            timeout (Optional[float]): Specifies the maximum amount of time in
+                seconds to wait for the measurement or generation to complete.
+                This method returns an error if the time elapses. The
+                default is 10. If you set timeout (sec) to
+                nidaqmx.WAIT_INFINITELY, the method waits indefinitely. If you
+                set timeout (sec) to 0, the method checks once and returns
+                an error if the measurement or generation is not done.
+        """
+        self._interpreter.wait_for_valid_timestamp(self._handle, timestamp_event, timeout)
```

Comment:
The interpreter methods expect ints, not enums. A test would have caught this.
```suggestion
        self._interpreter.wait_for_valid_timestamp(self._handle, timestamp_event.value, timeout)
```